HomePage.html

```diff
@@ -22,56 +22,57 @@
 			<img src="Assets/Icons/menu.svg" alt="Menu Icon" class="menu-icon" />
 		</header>
 
+		<!-- header for large screens -->
+		<header class="homepage-header-large-screen">
+			<nav>
+				<a href="#portfolio">Portfolio</a>
+				<a href="#about-me">About</a>
+				<a href="#contact">Contact</a>
+			</nav>
+		</header>
+		
 		<section class="homepage-section">
+			
 			<article>
-				<h1 class="homepage-h1">Hey there. I'm Wamae <br> <span>I'm a software developer </span> </h1>
+				<h1 class="homepage-h1">Hey there. I'm Wamae. <br> <span>I'm a software developer </span> </h1>
 				<p> I can help you build a product, feature or website. Look through some of my work and experience! If you like
 					what you see and have a project you need coded,
 					don’t hesitate to contact me.</p>
 			</article>
+		
 			<ul class="social-media-list">
-				<li>
-					<a href="">
-						<img src="Assets/Icons/twitter.svg" alt="Twitter" />
-					</a>
+				<li><a href="">
+						<img src="Assets/Icons/twitter.svg" alt="Twitter" /></a>
 				</li>
 
-				<li>
-					<a href="">
-						<img src="Assets/Icons/Linkedin-icon.svg" alt="Linked In" />
-					</a>
+				<li><a href="">
+						<img src="Assets/Icons/Linkedin-icon.svg" alt="Linked In" /></a>
 				</li>
 
-				<li>
-					<a href="">
-						<img src="Assets/Icons/medium.svg" alt="Medium" />
-					</a>
+				<li><a href="">
+						<img src="Assets/Icons/medium.svg" alt="Medium" /></a>
 				</li>
 
-				<li>
-					<a href="">
-						<img src="Assets/Icons/github.svg" alt="Github" />
-					</a>
+				<li><a href="">
+						<img src="Assets/Icons/github.svg" alt="Github" /></a>
 				</li>
 
-				<li>
-					<a href="">
-						<img src="Assets/Icons/angellist.svg" alt="Angel List" />
-					</a>
+				<li><a href="">
+						<img src="Assets/Icons/angellist.svg" alt="Angel List" /></a>
 				</li>
 
 			</ul>
 
 			<div class="scroll-button">
-				<a href="#projects"><img src="Assets/Icons/scroll-button.svg" alt="" /></a>
+				<a href="#portfolio"><img src="Assets/Icons/scroll-button.svg" alt="" /></a>
```

Comment:
- [Optional] Please consider using an indentation of two spaces in all your documents. This is a convention and also[ best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)